### ISSUE TITLE
Downloading collections

### ISSUE THREAD

**Issue by ylluminate** [wordtreefoundation/archdown#2]
I'd like to use this tool to download entire member upload collections for datasets. Ie, `https://archive.org/details/someuser`. Does this tool already support this functionality? From some of your examples it seems as though it might, but it's lacking in clarity as to how to do this.

**Comment by canadaduane** [comment]
Do you know what the filter would look like, using archive.org's syntax, to hone in on the collection you want? Archdown uses `archivist-client` which supports pretty much any filter (I think). See https://github.com/wordtreefoundation/archivist-client/blob/62fff6430d77d30602c7ec9dd1ffb1352a6aba78/lib/archivist/client/filters.rb

**Comment by ylluminate** [comment]
So you're saying it's not as simple as saying grab everything hanging off of this URL, but rather that the API must be used to "filter" content in order to pull in an uploader's uploads... right?

**Comment by canadaduane** [comment]
Yes, as far as I understand it, the URLs at archive.org don't map directly
to their query API. So the first step is to figure out how they are
querying/filtering to get that result, and then replicate it using
something like Archdown.
